examples/graphql-api/app/utils/index.ts

```diff
@@ -0,0 +1,27 @@
+import { GRAPHQL_API } from "~/config";
+
+/**
+ * @name fetchFromGraphQL
+ * @external https://css-tricks.com/raw-graphql-querying
+ * @description This function is used to fetch data from the GraphQL API.
+ * Check out the link above for more information.
+ */
+ export const fetchFromGraphQL = async (
+    operation: 'mutation' | 'query',
```

Comment:
The GraphQL query/mutation/subscription should always be provided via the `query` body property.

```suggestion
```

Reply:
Doh, thanks 👍